firmware/application/apps/ui_playlist.hpp

```diff
@@ -119,10 +119,10 @@ class PlaylistView : public View {
         {10 * 8, 1 * 16, 7 * 8, 16}};
 
     ProgressBar progressbar_track{
-        {18 * 8, 1 * 16, 12 * 8, 8}};
+        {18 * 8, 1 * 16, 12 * 8, 8 + 1}};
```

Comment:
Why make the top one bigger?

Reply:
Thank you for pointing this out! :) I wrote these two progress bar before, then this time, i need to make there's 1px line (instead of two) between two progress bars, by letting  1px overlapped. So, since they overlapped 1px, they are visually same, and compared before, it looks better. 
Before:
![image](https://github.com/eried/portapack-mayhem/assets/24917424/12b82afe-a3c1-4aad-abc7-43a814620ce3)
After:
![IMG_0542](https://github.com/eried/portapack-mayhem/assets/24917424/e038abe7-9978-4c54-8981-219bc3d9e10f)